Entity/User.php

```diff
@@ -199,4 +199,16 @@ public function isEnabled();
      * @return string
      */
     public function getUsername();
+
```

Comment:
Entity/User is now the User object from LocalUserBundle . The interface is renamed into Entity/UserInterface

Reply:
The branch has been rebased

Reply:
We see all the commits of develop in your PR, I think you needed to "merge" to not rewrite all the commit hashes!